frontend/amundsen_application/api/search/v0.py

```diff
@@ -40,6 +40,7 @@ def search_table() -> Response:
     Parse the request arguments and call the helper method to execute a table search
     :return: a Response created with the results from the helper method
     """
+    results_dict = {}
```

Comment:
The exception handled here https://github.com/amundsen-io/amundsen/blob/bd16c15763438a97a90688d1f516132fed40d2d8/frontend/amundsen_application/api/search/v0.py#L62
could be caused by calling `jsonify` here https://github.com/amundsen-io/amundsen/blob/bd16c15763438a97a90688d1f516132fed40d2d8/frontend/amundsen_application/api/search/v0.py#L58 and so you will encounter the same problem.